### PR TITLE
Day2

### PR DESCRIPTION
**Pull request by JoseGC789**
El pull anterior creo que lo hice mal.

### CODE REVIEW

File: day2/src/blog/BlogMain.java [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+        SelectActionOptions options = SelectActionOptions.values()[(enterInput(1,4)-1)];
+        switch (options){
+            case POST:
+                postEntry();
+                break;
+            case DELETE:
+                deleteEntry();
+                break;
+            case SHOW10:
+                showRecent10();
```
Comment (sebastianscatularo) [inline day2/src/blog/BlogMain.java]: decouple the 10 entries, what happens if the user wants 50 instead of 10?

File: day2/src/blog/BlogMain.java [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+                break;
+            case EXIT:
+                return false;
+        }
+        return true;
+    }
+
+    private void showRecent10 (){
+        //show most recent 10 entries to the user
+        System.out.printf("Most recent 10 entries: \n");
```
Comment (sebastianscatularo) [inline day2/src/blog/BlogMain.java]: Try creating a Writer / Printer class to decouple from standard out

File: day2/src/blog/BlogMain.java [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+        Entry aux = null;
+        for (Entry entry: entries){
+            //ID select
+            if (entry.getId()==id){
+                aux = entry;
+            }
+        }
+        //Deleting entry if found
+        //aux may have the entry otherwise it will just be null
+        //IMPORTANT: an entry cannot be deleted in the for loop else java.util.ConcurrentModificationException is thrown
```
Comment (sebastianscatularo) [inline day2/src/blog/BlogMain.java]: you could use an iterator to archive this

File: day2/src/blog/Entry.java [hunk 4]
```diff
@@ -0,0 +1,34 @@
+package blog;
+
+import java.util.Date;
+import java.util.Set;
+
+public class Entry {
+    private final int id; //KEY
+    private final String title;
+    private final String text;
+    private final Date date;
```
Comment (sebastianscatularo) [inline day2/src/blog/Entry.java]: getters?

Comment (JoseGC789) [reply day2/src/blog/Entry.java]: Will do

File: day2/src/blog/BlogMain.java (hunk 3 above)
Comment (JoseGC789) [reply day2/src/blog/BlogMain.java]: I reworked the iterator.

File: day2/src/blog/BlogMain.java (hunk 1 above)
Comment (JoseGC789) [reply day2/src/blog/BlogMain.java]: oh the decoupling you mentioned is fully implemented in day3 pr

**Pull request merged**